MeretzLib/Apple/iOS/Meretz/Meretz/Meretz.h

```diff
@@ -138,9 +183,20 @@ typedef NS_ENUM(NSInteger, MeretzTaskStatus)
 	// Item consumption over a date range
 	// startDate: date to use for start of the search
 	// endDate: date to use for the end of the search (if nil, current day is assumed)
-	- (MeretzTaskId) vendorConsume: (NSDate *) startDate optional: (NSDate *) endDate;
```

Comment:
vendorConsume is renamed to vendorConsumeWithinRange, and then a few new APIs were added:
* vendorConsumeGetNew (GET /vendor/consume)
* vendorConsumeAcknowledge (PUT /vendor/consume with an array of Item public_ids)
* vendorConsumeGetNewAndAcknowledge (PUT /vendor/consume with an empty array of Item public_ids - meaning, consume everything new)